### PR TITLE
Add cli

### PR DESCRIPTION
**Pull request by Arthelon**
Addresses #57 

Blocked by the publishing of `@oursky/gitlint`

### CODE REVIEW

File: packages/cli/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  .command(
+    ["lint", "*"],
+    'Lints commits in git repository. Reads last commit message from ".git/COMMIT_EDITMSG" by default',
+    {
+      config: {
+        alias: "c",
+        description: "path to config file",
+        type: "string",
+      },
+      from: {
```
Comment (kiootic) [inline packages/cli/src/index.ts:35]: The standard way to represent a git commit range is to use dotted range syntax: https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection

Can we support that as well?

File: packages/cli/src/commands/lint/index.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    messages.map(async (message) => {
+      const result = await lintCommitMessage(message, preset);
+      return {
+        ...result,
+        commitMessage: message,
+      };
+    })
+  );
+  const formattedResults = formatResults(results);
+  if (formattedResults !== "") {
```
Comment (kiootic) [inline packages/cli/src/commands/lint/index.ts:68]: Result formatting is a presentation concern, the behavior of the command line tools should not depends on the output format (e.g. we may want to print a statement when there is no violations).

I suppose we can always print the formatted result, and the exit code would depends on whether there is violations in any of the commit messages.

File: packages/cli/src/index.ts (hunk 1 above)
Comment (Arthelon) [reply packages/cli/src/index.ts:35]: yes but we can only support the double-dot ranges

Comment (kiootic) [reply packages/cli/src/index.ts:35]: Sure, I think that's okay.

**Pull request merged**